lib/puppet/provider/dsc_base_provider/dsc_base_provider.rb

```diff
@@ -79,6 +79,11 @@ def canonicalize(context, resources)
             end
             downcased_result = recursively_downcase(canonicalized)
             downcased_resource = recursively_downcase(r)
+
```

Comment:
maybe remove this blank line and the one below to adhere to the format of the rest of the file?